packages/next/src/build/webpack/loaders/next-font-loader/postcss-next-font.ts

```diff
@@ -45,8 +43,7 @@ const postcssNextFontPlugin = ({
       }
 
       const formatFamily = (family: string) => {
-        // Turn the font family unguessable to make it locally scoped
-        return `'__${family.replace(/ /g, '_')}_${fontFamilyHash}'`
+        return `'${family}'`
```

Comment:
It wraps the family with the quotes in case the font name itself is using any special characters that would break the css property assignment when used without quotes